GetSchwifty/Code Helper/FormattingHelper.swift

```diff
@@ -203,6 +203,7 @@ private extension UITextView {
     
     // MARK: - ACTIONS
     func newLine(_ times: UInt = 1) {
+        guard times > 0 else { return }
```

Comment:
`times` is an `UInt`, so if the given number is < 0 it would creash on function call of newLine